src/registry/billingsdk/demo/billing-settings-2-demo.tsx

```
@@ -0,0 +1,217 @@
+"use client"
+
+import { useState } from "react"
+import { Dialog, DialogContent, DialogHeader, DialogTitle } from "@/components/ui/dialog"
+import { Button } from "@/components/ui/button"
+import { Input } from "@/components/ui/input"
+import { Label } from "@/components/ui/label"
+import { useState } from "react"
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useState'. Consider to delete it or rename it.

'useState' is defined here:


(lint/suspicious/noRedeclare)